libs/cdk/zone-configurations/src/lib/zone-config.spec.ts

```diff
@@ -19,10 +19,6 @@ describe('zone-config', () => {
     .filter((property) => property.toLowerCase().startsWith('__zone'))
     .map((property) => [property, w[property]]);
 
-  beforeAll(() => {
-    w.Zone = undefined;
-  });
```

Comment:
Note: with the new update, for a reason I ignore, I had to remove this line otherwise all tests fail with:

```
TypeError: Cannot read properties of undefined (reading 'current')

      at Function.Object.<anonymous>.ProxyZoneSpec.get (../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:3737:25)
      at Function.Object.<anonymous>.ProxyZoneSpec.isLoaded (../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:3740:34)
      at Function.Object.<anonymous>.ProxyZoneSpec.assertPresent (../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:3743:32)
      at Object.resetFakeAsyncZone (../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:5588:54)
      at resetFakeAsyncZone (../../node_modules/@angular/core/fesm2015/testing.mjs:272:36)
      at ../../node_modules/@angular/core/fesm2015/testing.mjs:26809:13
      at _ZoneDelegate.Object.<anonymous>._ZoneDelegate.invoke (../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:409:30)
      at ProxyZoneSpec.Object.<anonymous>.ProxyZoneSpec.onInvoke (../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:3830:43)
      at _ZoneDelegate.Object.<anonymous>._ZoneDelegate.invoke (../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:408:56)
      at Zone.Object.<anonymous>.Zone.run (../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:169:47)
      at Object.wrappedFunc (../../node_modules/zone.js/bundles/zone-testing-bundle.umd.js:4330:34)
```

I'm not sure why we need Zone to be defined in the test environment as zone-flags should be imported **before** Zone.js, it's maybe a problem with the zone-testing module itself as it was not happening before the update.